controllers/front/CartController.php

```diff
@@ -637,7 +637,7 @@ protected function areProductsAvailable()
         if ($product['active']) {
             return $this->trans(
                 'The item %product% in your cart is no longer available in this quantity. You cannot proceed with your order until the quantity is adjusted.',
```

Comment:
```suggestion
                'The product %product% in your cart is no longer available in this quantity. Please adjust the quantity to proceed with the order.',
```
This wording has been changed in another PR (https://github.com/PrestaShop/PrestaShop/pull/23093/files#diff-0dac05a6abe4d202319dd093b8ab2b37R654)